main.py

```diff
@@ -253,7 +256,15 @@ def event_create(self, event):
         error = False
 
         try:
-            box = Box(event.content['data'], event.content['namespace'])
+            data = event.content['data']
+            namespace = event.content['namespace']
+            box_id = event.content.get('box_id')
+
+            results = self.search_metadata_by(namespace, query=box_id)
+            if results:
+                raise KeyError
```

Comment:
Shouldn't we give a message for KeyError? `KeyError('some message about the error')`